src/main/java/CovidBot.java

```diff
@@ -26,11 +26,11 @@ public static void main(String[] args) {
                     isInteger = 0;
                 }
 
-                if(isInteger == 1) {
+                if (isInteger == 1) {
                     int d = Integer.parseInt(checkDone[1]);
-                    toDos[d-1].markAsDone();
+                    toDos[d - 1].markAsDone();
                     System.out.println("Good job! :) I've marked this task as complete:\n");
```

Comment:
You dont have to use 4 back to back System.out.println. The use of /n actually creates a new print on the next line.

Reply:
Hello! I've edited my code and put the print lines that I could into one line, thanks for the advice! :)